packages/solid/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/solid",
-  "version": "2.0.0",
+  "version": "2.0.1-alpha.1",
```

Comment:
Perfect!
Please also bump the version of @web3-onboard/core on line 66 below to match the new core package alpha version

Reply:
Thank you! I've also bumped the version in https://github.com/blocknative/web3-onboard/pull/2057/commits/fc3c8316025f6b486bc13e83ea11713e28bb3f44